.github/release-drafter.yml

```diff
@@ -0,0 +1,42 @@
+
+name-template: 'v$NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: 'API'
+    label: 'API'
+  - title: 'Examples'
+    label: 'examples'
+  - title: 'Documentation'
+    label: 'documentation'
+  - title: 'Compiler'
+    label: 'compiler'
+  - title: 'MPI'
```

Comment:
Do we need MPI, GPU, arch, bug-C/performance, benchmarking categories?
I feel that those are specific to Devito, and do not make much sense for xDSL?

Reply:
Definitely agree, this was just a sketch